src/silvimetric/commands/scan.py

```diff
@@ -1,23 +1,103 @@
 import numpy as np
 import logging
+import dask.bag as db
+import dask
+import math
 
-from ..resources import Storage, Data, Extents, Log
+from ..resources import Storage, Data, Extents, Bounds
 
-def scan(tdb_dir, pointcloud, bounds, point_count, resolution):
+def scan(tdb_dir, pointcloud, bounds, point_count=600000, resolution=100, depth=6):
```

Comment:
where do these magic numbers come from?

Reply:
They're just magic numbers that seem to work well, and can be modified by the user if they want.